library/core/src/slice/mod.rs

```diff
@@ -1778,19 +1778,24 @@ impl<T> [T] {
     /// # Examples
     ///
     /// ```
-    /// #![feature(slice_strip)]
     /// let v = &[10, 40, 30];
     /// assert_eq!(v.strip_prefix(&[10]), Some(&[40, 30][..]));
     /// assert_eq!(v.strip_prefix(&[10, 40]), Some(&[30][..]));
     /// assert_eq!(v.strip_prefix(&[50]), None);
     /// assert_eq!(v.strip_prefix(&[10, 50]), None);
+    ///
+    /// let prefix : &str = "he";
+    /// assert_eq!(b"hello".strip_prefix(prefix.as_bytes()),
+    ///            Some(b"llo".as_ref()));
     /// ```
     #[must_use = "returns the subslice without modifying the original"]
-    #[unstable(feature = "slice_strip", issue = "73413")]
-    pub fn strip_prefix(&self, prefix: &[T]) -> Option<&[T]>
+    #[stable(feature = "slice_strip", since = "1.50.0")]
```

Comment:
This should be 1.51 right ?

Reply:
Err, perhaps.  I have just tried looking for git tags or something to tell me which Rust versions contain 8f0b945cfcc3d084583bc27a7ed23b27b1246751, the relevant commit, but there don't seem to be any tags for beta versions.  I guess that makes sense.  I found that `origin/beta` does not contain that commit, and the forge tells me beta is 1.50.  So should I send a MR to fix this ?

Reply:
This PR is on 1.51.0 milestone according to rustbot.
Yes, someone could send another PR to update the stabilization version.